railties/test/commands/routes_test.rb

```diff
@@ -197,7 +197,6 @@ class Rails::Command::RoutesTest < ActiveSupport::TestCase
       end
     RUBY
 
-    # rubocop:disable Layout/TrailingWhitespace
```

Comment:
The test failures demonstrate why we need this.

Reply:
Ah, I see. I returned it back for now.